include/exadg/poisson/overset_grids/driver.cpp

```diff
@@ -50,11 +50,31 @@ Driver<dim, n_components, Number>::setup()
 {
   pcout << std::endl << "Setting up Poisson solver for overset grids:" << std::endl;
 
-  application->setup();
+  // setup application
+  AssertThrow(application->domain1.get(), dealii::ExcMessage("Domain 1 is uninitialized."));
```

Comment:
this is the content of the deleted `application->setup` function, right? .. containing the `set_boundary_ids()`, e.g., which you now moved here directly? Deleting the definitions in `include/exadg/poisson/overset_grids/user_interface/application_base.h`.